.github/workflows/ci-rust.yml

```diff
@@ -106,6 +106,10 @@ jobs:
         env:
           RUST_LOG: debug
 
+      - name: Test platform-async on wasm
```

Comment:
You plan on having another PR to put this behind a check to only run if platform crates have been modified ? (in order to avoid wasm compilation when not needed)

Reply:
I've created the issue #5180